cellarium/ml/core/module.py

```diff
@@ -329,15 +329,9 @@ def on_train_epoch_start(self) -> None:
             if callable(set_epoch):
                 set_epoch(self.current_epoch)
 
-    def on_train_start(self) -> None:
-        """
-        Calls the ``on_train_start`` method on the :attr:`model` attribute.
-        If the :attr:`model` attribute has ``on_train_start`` method defined, then
-        ``on_train_start`` must be called at the beginning of training.
-        """
-        on_train_start = getattr(self.model, "on_train_start", None)
-        if callable(on_train_start):
-            on_train_start(self.trainer)
+        on_train_epoch_start = getattr(self.model, "on_train_epoch_start", None)
+        if callable(on_train_epoch_start):
```

Comment:
Would be nice to somehow call the delegate_method here so if there are any changes in it this wouldn't need any changes.

Reply:
I am not sure if I understand this point.  I was thinking that we could not re-implement the explicitly implemented methods (like `on_train_epoch_start`) because the passthrough/delegate would end up having the same name and they would conflict (I don't know how python handles that).